### PR TITLE
Databases-week3

### PR DESCRIPTION
**Pull request by patel-prakashkumar**
I have added assignment database week3.

### CODE REVIEW

File: databases/week3/Assignmnet_Task.sql [hunk 1]
```diff
@@ -0,0 +1,104 @@
+-- Active: 1660903523654@@127.0.0.1@3306@database_week3
+USE database_week3;
+-- Queries : Meal TABLE
+SELECT * FROM `Meal`;
```
Comment (Elia-Git) [inline databases/week3/Assignmnet_Task.sql]: Remove parenthesis **SELECT * FROM `Meal`;** around table names. It doesn't run on vs code. it should be like this `SELECT * FROM Meal;` This applies to all codes of the line where you put parenthesis around table names.

File: databases/week3/Assignmnet_Task.sql [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+WHERE meal.title = 'Breakfast'
+ORDER BY created_date;
+-- Sort all meals by average number of stars in the reviews
+SELECT
+meal.title,
+Round(AVG(Review.stars), 0) As Average
+FROM `Meal`
+INNER JOIN Review ON meal.id = Review.meal_id
+GROUP BY Review.meal_id
+ORDER BY AVG(Review.stars);
```
Comment (Elia-Git) [inline databases/week3/Assignmnet_Task.sql]: Pls pay attention to try if your queries run correctly (syntax wise) before pushing to GH.

File: databases/week3/Craete_Databse.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+)ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `Review` (
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `title` VARCHAR(255) NOT NULL,
+    `description` TEXT NULL DEFAULT NULL,
+    `meal_id` INT(10) UNSIGNED NOT NULL,
+    `stars` INT(10) UNSIGNED NOT NULL,
+    `created_date` DATE ,
+    CONSTRAINT `fk_review_meal` FOREIGN KEY (`meal_id`) REFERENCES `Meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
```
Comment (Elia-Git) [inline databases/week3/Craete_Databse.sql]: Good job here! We use parenthesis here when you create the name, but not when quering.

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Pls, modify the whole file(.../Assignment_Task) file by removing all parenthesis around the table names. Check how your queries were in the last 2 modules.
Double-check your code if it works before pushing.
_Good ER diagram_

File: databases/week3/Assignmnet_Task.sql (hunk 1 above)
Comment (patel-prakashkumar) [reply databases/week3/Assignmnet_Task.sql]: I will remove backtit `` .  Thank you for feedback

File: databases/week3/Craete_Databse.sql (hunk 3 above)
Comment (patel-prakashkumar) [reply databases/week3/Craete_Databse.sql]: ahmm you are right, I was confuse some of the sql file I have seen backtit `` in table name but some of the I couldn't.

**Pull request merged**